<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="2.96s"
		durationms="2963"
		type="root"
		rid="15dde1e20e341c76"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="erdogb"
		rxversion="8.1.1.0"
		host="EC168"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="de-DE"
		dtnlanguage="de-CH"
		screenresolution="3840x1200"
		timestamp="23.05.2018 12:53:29"
		timestampiso="2018-05-23T10:53:29"
		endtime="23.05.2018 12:53:32"
		timeoutfactor="1"
		module="Analysis">
		<detail>
			Analysis
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:02.963"
			timeRelativeToTestModuleStartTime="00:02.964"
			timeWallClock="12:53:32"
			time="00:02.963"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
